packages/plugin-types/src/deep_partial.ts

```diff
@@ -5,3 +5,17 @@
 export type DeepPartial<T> = {
   [P in keyof T]?: DeepPartial<T[P]>;
 };
```

Comment:
I presume `DeepPartial` is no longer needed?

Reply:
Yeah it is not being used anywhere else at the moment, I'll remove it.